controllers/awsmachine_controller.go

```diff
@@ -85,7 +80,7 @@ func (r *AWSMachineReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, reter
 		return reconcile.Result{}, nil
 	}
 
-	logger = logger.WithName(fmt.Sprintf("machine=%s", machine.Name))
+	logger = logger.WithValues("machine", fmt.Sprintf("%s/%s", machine.Namespace, machine.Name))
```

Comment:
From here and blow we can likely leave off the namespace, since they will be the same as the `AWSMachine` above.